### PR TITLE
[Bug][Function] fix lag/lead function get invalid data

### LINKED ISSUE #9033: [Bug] window function `lag` return invalid data
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

1.0.0-rc03

### What's Wrong?

```
CREATE TABLE `wftest` ( `aa` varchar(10) NULL COMMENT "", `bb` text NULL COMMENT "", `cc` text NULL COMMENT "" ) ENGINE=OLAP UNIQUE KEY(`aa`) DISTRIBUTED BY HASH(`aa`) BUCKETS 3 PROPERTIES ( "replication_allocation" = "tag.location.default: 1", "in_memory" = "false", "storage_format" = "V2" );

insert into wftest values
('a','aa','/wyyt-image/2021/11/13/595345040188712460.jpg'),
('b','aa','/wyyt-image/2022/04/13/1434607674511761493.jpg'),
('c','cc','/wyyt-image/2022/04/13/1434607674511761493.jpg') 

mysql> select min(t.cc) over(PARTITION by t.cc  order by t.aa) ,lag(t.cc,1,'')over(PARTITION by t.cc  order by t.aa) as l1 from wftest t;
+------------------------------------------------+------------------------------------------------+
| min(`t`.`cc`) OVER(...)                        | l1                                             |
+------------------------------------------------+------------------------------------------------+
| /wyyt-image/2021/11/13/595345040188712460.jpg  |                                                |
| /wyyt-image/2022/04/13/1434607674511761493.jpg |                                                |
| /wyyt-image/2022/04/13/1434607674511761493.jpg |                 ` �7  .       � �7           |
+------------------------------------------------+------------------------------------------------+
```

column `l1`'s result is wrong.

This query works well when `set enable_vectorized_engine=true`;

### What You Expected?

valid result

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes

Issue Number: close #9033 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyticExpr.java [hunk 1]
```diff
@@ -478,11 +478,13 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
 
         // min/max is not currently supported on sliding windows (i.e. start bound is not
         // unbounded).
-        if (window != null && isMinMax(fn) &&
```
Comment (HappenLee) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyticExpr.java:481]: add comment here

Comment (zhangstar333) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyticExpr.java:481]: done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**